### ISSUE TITLE
droneaid-counter.js doesn't actually count

### ISSUE THREAD

**Issue by johnwalicki** [Code-and-Response/DroneAid#17]
**Describe the bug**
The model detects one of the classified images and draws a bounding box but the counter on the web page does not increment

**To Reproduce**
1. View some of the symbol images.
2. Observe that the model draws bounding boxes.
3. The web page does not increment

**Comment by krook** [comment]
@pedrocruzio Thoughts on this one?

**Comment by johnwalicki** [comment]
Va and I had a chat about this ticket in November.  Essentially, it doesn't increment the counter if the prior frames bounding boxes intersect the current bounding box.  Which makes sense, the drone found a symbol and is just moving around it.  You don't want to count it twice (or hundreds of times). 

I still think its fickle and not quite working as expected.

**Comment by krook** [comment]
@johnwalicki @vabarbosa do we want to keep this issue open?

**Comment by vabarbosa** [comment]
yes, please keep open. there are still some additional updates i would like to make
